app/views/donors/_show.html.slim

```diff
@@ -17,7 +17,7 @@
 
           .col-xs-6.col-md-6
             p <b>Address:</b> #{@donor.address} #{@donor.postal_code}
-            - if !@donor.postal_code&.empty? && Donor.same_postcode(@donor.postal_code).count > 1
+            - if @donor.postal_code&.present? && Donor.same_postcode(@donor.postal_code).count > 1
```

Comment:
Awesome! I think we can move the part that excludes this user them self into the model method, and keep the other two branches as they are. 😀